assignment/README.md

```diff
@@ -53,13 +64,13 @@ most difficult about this project? How will you attempt to cope with
 these difficulties? For example, asynchronous behavior (ajax, events)
 are hard to use and think about. Global variables are a strategy for
 coping with that difficulty by breaking data out of the asynchronous
-context.
+context.  
+__Data__:  
+Generating the data raster/vector data may be time-consuming.  
+How to load georeferenced geoTiff images onto the map, while preserving the accuracy and making it zoomable? I'll try different plugins to solve this problem.  
```

Comment:
You can make it zoomable by saving higher definition versions and choosing which to display based on zoom level, if you like